x/oracle/types/params.go

```diff
@@ -217,6 +217,10 @@ func (p Params) Validate() error {
 		return fmt.Errorf("oracle parameter SlashWindow must be greater than or equal with VotePeriod")
 	}
 
+	if p.SlashWindow%p.VotePeriod != 0 {
```

Comment:
```suggestion
	if p.SlashWindow % p.VotePeriod != 0 {
```

Reply:
Linter is against this for some reason so switching it back